### PR TITLE
Create jquery-trap-input.travis.yml

### PR DESCRIPTION
**Pull request by gururajrkatti**

### CODE REVIEW

**Comment by Siddhesh-Ghadi** [comment]
Since the `travis.yml` is created from scratch, should we also test it on `latest` and `lts` node versions or stick to one specfied in build-script? @seth-priya 

Rest all looks good.

**Comment by seth-priya** [comment]
yes, testing with latest versions make sense @Siddhesh-Ghadi

File: travis-ymls/jquery-trap-input.travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+
+language: nodejs
+
+dist: focal
+
+nodejs: 10
```
Comment (Siddhesh-Ghadi) [inline travis-ymls/jquery-trap-input.travis.yml:17]:
```suggestion
nodejs: 
    - 10
    - node  #latest stable Node.js release
    - lts/* #latest LTS Node.js release
```

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]
For all node `.travis.yml` which we create from scratch, lets test them on latest node versions also.

**Comment by gururajrkatti** [comment]
Thanks @Siddhesh-Ghadi, incorporated your changes, tested on travis, updated the travis log location in the file and squashed the commits.

**Comment by Siddhesh-Ghadi** [comment]
Ready for merge @seth-priya

**Pull request merged**